### PR TITLE
statistics: Add a note about the solution of a potential OOM issue

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7359
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: statistics.md [hunk 1]
```diff
@@ -6,7 +6,29 @@ aliases: ['/docs/dev/statistics/','/docs/dev/reference/performance/statistics/']
 
 # Introduction to Statistics
 
-TiDB uses statistics to decide [which index to choose](/choose-index.md). The `tidb_analyze_version` variable controls the statistics collected by TiDB. Currently, two versions of statistics are supported: `tidb_analyze_version = 1` and `tidb_analyze_version = 2`. In versions before v5.1.0, the default value of this variable is `1`. In v5.1.0, the default value of this variable is `2`, which serves as an experimental feature. These two versions include different information in TiDB:
+TiDB uses statistics to decide [which index to choose](/choose-index.md). The `tidb_analyze_version` variable controls the statistics collected by TiDB. Currently, two versions of statistics are supported: `tidb_analyze_version = 1` and `tidb_analyze_version = 2`. In versions before v5.1.0, the default value of this variable is `1`. In v5.1 and v5.2, the default value of this variable is `2`, which serves as an experimental feature.
```
Comment (qiancai) [inline statistics.md:9]:
```suggestion
TiDB uses statistics to decide [which index to choose](/choose-index.md). The `tidb_analyze_version` variable controls the statistics collected by TiDB. Currently, two versions of statistics are supported: `tidb_analyze_version = 1` and `tidb_analyze_version = 2`. In versions before v5.1.0, the default value of this variable is `1`. In v5.1, v5.2, and v5.3, the default value of this variable is `2`, which serves as an experimental feature.
```

File: statistics.md [hunk 2]
```diff
@@ -6,7 +6,29 @@ aliases: ['/docs/dev/statistics/','/docs/dev/reference/performance/statistics/']
...
+
+> **Note:**
+>
+> When `tidb_analyze_version = 2`, if memory overflow occurs after you execute `ANALYZE`, you need to set `tidb_analyze_version = 1` and perform one of the following operations:
```
Comment (qiancai) [inline statistics.md:13]:
```suggestion
> When `tidb_analyze_version = 2`, if memory overflow occurs after `ANALYZE` is executed, you need to set `tidb_analyze_version = 1` and perform one of the following operations:
```

Comment (qiancai) [reply statistics.md:13]: 不一定是用户执行的，可能是系统自动执行的，所以这里不能写 you execute

**Review by qiancai — COMMENTED** [review]
@TomShawn PTAL

File: statistics.md (hunk 1 above)
Comment (qiancai) [reply statistics.md:9]: 中文也请帮忙改下，谢谢

Comment (TomShawn) [reply statistics.md]: 已在 https://github.com/pingcap/docs-cn/pull/7884 中修改

**Review by qiancai — APPROVED** [review]

**Pull request merged**